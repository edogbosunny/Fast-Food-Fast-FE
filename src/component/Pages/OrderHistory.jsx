/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';
import OrderHistoryTableComponent from '../UI/tables/orderHistoryTable/OrderHistoryTable';

class OrderHistory extends Component {
  render() {
    return (
      <div className='order-history-container'>
        <table>
          <thead className='tbg table-text-color'>
            <tr>
              <th>Order-Id</th>
              <th>Ordered item(s)</th>
              <th>Cost</th>
              <th>Date Ordered</th>
              <th>Status</th>
            </tr>
          </thead>
          <tbody>
            <OrderHistoryTableComponent />
          </tbody>
        </table>
      </div>
    );
  }
}

export default OrderHistory;
