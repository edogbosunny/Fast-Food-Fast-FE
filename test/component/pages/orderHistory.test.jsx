import React from 'react';
import { shallow } from 'enzyme';
import OrderHistory from '../../../src/component/Pages/OrderHistory';

describe('Login page', () => {
  const wrapper = shallow(
    <OrderHistory />,
  );
  it('should mount the login page', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
