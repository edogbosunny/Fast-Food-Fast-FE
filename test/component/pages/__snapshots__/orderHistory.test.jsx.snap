// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page should mount the login page 1`] = `
<div
  className="order-history-container"
>
  <table>
    <thead
      className="tbg table-text-color"
    >
      <tr>
        <th>
          Order-Id
        </th>
        <th>
          Ordered item(s)
        </th>
        <th>
          Cost
        </th>
        <th>
          Date Ordered
        </th>
        <th>
          Status
        </th>
      </tr>
    </thead>
    <tbody>
      <Connect(OrderHistoryTable) />
    </tbody>
  </table>
</div>
`;
